core/chapters/c12_dictionaries.py

```diff
@@ -425,6 +432,10 @@ def total_cost(quantities: Dict[str, str], prices: Dict[str, str]):
 
         hints = "TODO PPPP"
 
+        # Custom generate inputs is required because cart and prices must have the same keys.
+        def generate_inputs():
+            return {'quantities': {"apple" : 2, "banana" : 5}, 'prices': {"apple" : 7, "banana" : 11}}
```

Comment:
`generate_inputs` should generate values that fit the required constraints but are otherwise random and will usually be different each time it's called. When the user's submission is checked, `generate_inputs` is called 10 times to create different test cases. This ensures that the user has really found a general solution and not just written a bunch of `if` statements that pass the initial tests. Static examples like this one belong in the `tests` list below.